media/static-bundles.json

```diff
@@ -1034,19 +1034,15 @@
     },
     {
       "files": [
-        "protocol/js/protocol-modal.js",
         "js/libs/spin.min.js",
-        "js/base/send-to-device.js",
```

Comment:
The S2D forms seem to be broken because you're removing the JS here?

Reply:
Yeesh, what the heck. Will add that back in. Thank you for the review, Alex.